### ISSUE TITLE
Getting different output when running tsc directly

### ISSUE THREAD

**Issue by glen-84** [ivogabe/gulp-typescript#292]
As the title says. Using the latest version (2.12.0).

Could it be something to do with the `allowSyntheticDefaultImports` compiler option in my tsconfig.json file?

This was working with the previous version + a custom TypeScript 1.8.0, so I'm not sure what has changed.

http://www.mergely.com/XTglqaoQ/ (direct left, gulp-ts right).

**Comment by ivogabe** [comment]
It looks like that `tsc` uses `module: "es6"` and gulp uses `module: "commonjs"`. Can you post your tsconfig and gulpfile?

`allowSyntheticDefaultImports` is only used for type information, so this shouldn't affect the emit.

**Comment by glen-84** [comment]
```json
{
    "compilerOptions": {
        "target": "es6",
        "allowSyntheticDefaultImports": true,
        "rootDir": "src",
        "outDir": "dist",
        "newLine": "LF"
    },
    "exclude": [
        ".git",
        "dist",
        "node_modules",
        "typings/browser",
        "typings/browser.d.ts"
    ],
    "compileOnSave": false
}
```

```js
import gulp from "gulp";
import ts from "gulp-typescript";
import babel from "gulp-babel";

// Build.
gulp.task("build", () => {
    let tsProject = ts.createProject("tsconfig.json", {
        //typescript: require("typescript")
    });

    return tsProject.src()
        .pipe(ts(tsProject))
        //.pipe(babel())
        .pipe(gulp.dest(tsProject.config.compilerOptions.outDir));
});
```

**Comment by glen-84** [comment]
@ivogabe Were you able to reproduce this? Do you need the source file?

**Comment by ivogabe** [comment]
I haven't had time to take a look at it yet. I hope to do that this weekend or next week. I think I have enough information to debug this. You can add `"module": "es6"` to the compiler options in the mean time, to fix this.

**Comment by glen-84** [comment]
Awesome! Thanks for the update.

**Comment by ivogabe** [comment]
Fixed in #297
